src/app/routes/settings.account.tsx

```
@@ -0,0 +1,65 @@
+import { useMe, useUpdateMe } from '@/features/auth/api/useAuth';
+import LabeledTextInput from '@/shared/ui/LabeledTextInput';
+import Layout from '@/shared/ui/layout/Layout';
+import SubPageHeader from '@/shared/ui/SubPageHeader';
+
+export const Route = createFileRoute('/settings/account')({
+  component: RouteComponent,
+});
+
+function RouteComponent() {
+  const updateMe = useUpdateMe();
+  const { data, isLoading, isError, error } = useMe();
```

ESLint

[error] 'isLoading' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)